govcd/system_test.go

```diff
@@ -233,3 +234,82 @@ func (vcd *TestVCD) Test_QueryOrgVdcNetworkByName(check *C) {
 	check.Assert(orgVdcNetwork[0].Name, Equals, vcd.config.VCD.Network.Net1)
 	check.Assert(orgVdcNetwork[0].ConnectedTo, Equals, vcd.config.VCD.EdgeGateway)
 }
+
+func (vcd *TestVCD) Test_QueryProviderVdcEntities(check *C) {
+	providerVdcName := vcd.config.VCD.ProviderVdc.Name
+	networkPoolName := vcd.config.VCD.ProviderVdc.NetworkPool
+	storageProfileName := vcd.config.VCD.ProviderVdc.StorageProfile
+	if providerVdcName == "" {
+		check.Skip("Skipping Provider VDC query: no provider VDC was given")
+	}
+	providerVdcs, err := vcd.client.QueryProviderVdcs()
+	check.Assert(err, IsNil)
+
+	testVerbose := os.Getenv("TEST_VERBOSE") != ""
```

Comment:
Thinking if we should make "global" for testing and then potentially use everywhere. Currently I guess we print out (or not) in "random" fashion.

Reply:
done